identity/app/views/consentJourney.scala.html

```diff
@@ -29,16 +31,28 @@
                     <div class="js-errorHolder manage-account__errors"></div>
 
                     <div class="manage-account-wizard manage-account-wizard--repermission" >
-                        <div class="manage-account-wizard__step">
-                            <h1>Welcome to repermissioning</h1>
-                        </div>
                         <div class="manage-account-wizard__step">
 
                             <form novalidate action="@idUrlBuilder.buildUrl("/privacy/edit", idRequest)" role="main" method="post">
                                 @views.html.helper.CSRF.formField
 
-                                <h2 class="manage-account-header">We need you to consent again to receiving communications from The Guardian</h2>
-                                <p class="manage-account-headerNote">These are all the permissions you already consented to, please review them and check them again if you still want to receive them</p>
+                                @if(journey == "repermission") {
```

Comment:
@mario-galic @Calum-Campbell i'm really unhappy abput this being a string. is there a formal way to have a more "clean" list of all possible journeys and validate them before?

Reply:
I think this is ok like this. We could have a list of acceptable values or even a journey type that we convert to but I think it's overkill.

Reply:
A sealed trait or an enum?

Reply:
[Example](https://github.com/guardian/frontend/blob/fa8306c64833c2479a56f0112157ccdbc1c301d3/identity/app/views/support/fragment/Switch.scala#L6)